src/scanners/javascript.js

```diff
@@ -19,22 +21,26 @@ export default class JavaScriptScanner {
     ensureFilenameExists(this.filename);
   }
 
-  scan(_ESLint=ESLint) {
+  scan(_ESLint=ESLint, _rules=this._defaultRules,
```

Comment:
I think once we get to this it's probably time to consider using an object. Makes it super-obvious from the calling side too.